Packages/com.bci4kids.bciessentials/Runtime/Scripts/Behaviors/BCIControllerBehavior.cs

```diff
@@ -19,54 +19,72 @@ public abstract class BCIControllerBehavior : MonoBehaviour
         /// The type of BCI behavior implemented.
         /// </summary>
         public abstract BCIBehaviorType BehaviorType { get; }
-
+        [Header("BCI Registration")]
         [SerializeField]
         [Tooltip("Register and Unregister with the BCI Controller instance using Start and OnDestroy")]
         private bool _selfRegister = true;
         
         [SerializeField]
         [Tooltip("Whether to set as active behavior when self registering.")]
         private bool _selfRegisterAsActive;
+
+        [Header("Default SPO Setup")]
+        [Tooltip("Component used to set-up default SPO objects")]
+        [SerializeField] protected MatrixSetup setup;
+        [Tooltip("Indicate whether or not to use the setup component")]
+        public bool setupRequired;
         
+        [Header("System Properties and Targets")]
         [SerializeField]
         [Tooltip("The applications target frame rate. 0 results in no override being applied. -1 or higher than 0 is still applied.")]
         [Min(-1)]
         protected int targetFrameRate = 60;
 
+        [SerializeField]
+        [Tooltip("Enable BCIController Hotkeys")]
+        public bool _hotkeysEnabled = true;
+
+        [Header("SPO Objects")]
         [SerializeField]
         [Tooltip("Provide an initial set of SPO.")]
         protected List<SPO> _selectableSPOs = new();
 
         private int __uniqueID = 1;
 
-        [SerializeField]
-        [Tooltip("Enable BCIController Hotkeys")]
-        public bool _hotkeysEnabled = true;
-
-
         #region Refactorable Properties
 
+        [Header("BCI Signal Properties")]
         //StimulusOn/Off + sending Markers
+        [Tooltip("The length of the processing window")]
+        //TODO: Rename this more appropriately to our Epoch/scheme
         public float windowLength = 1.0f;
+        [Tooltip("The interval between processing windows")]
         public float interWindowInterval = 0f;
 
         //Training
-        [SerializeField] protected MatrixSetup setup;
-        public bool setupRequired;
+        [Header("BCI Training Properties")]
+        [Tooltip("The number of training iterations")]
         public int numTrainingSelections;
+        [Tooltip("The number of windows used in each training iteration")]
```

Comment:
numTrainingSelections is the number of training trials. For example 6 selections does 6 trials. For example we have 3 objects, and 6 selections, then Bessy will train on each object twice.

Reply:
OOOOOPS disregard, I am just noticing that the tooltip is above the var it corresponds to.

Reply:
Actually, a note here that numTrainWindows doesn't do anything for P300, since the number of windows is controlled by number of flashes lower and upper limits. Could we add this to the tooltip or even hide it from the p300 controller behaviour inspector.

Reply:
I don't think I can hide it, as it's from the BCIControllerBehavior.cs base class...but I can include ont he tooltip that it doesn't affect P300!